addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/handler/BoxHandler.java

```diff
@@ -174,6 +185,21 @@ private void updateThingFromDevice(Thing thing, DeviceModel device) {
                 updateState(channelPower.getUID(), new DecimalType(device.getPowermeter().getPower()));
             }
             if (device.isSwitchableOutlet() && device.getSwitch() != null) {
+                Channel channelMode = thing.getChannel(CHANNEL_MODE);
+                if (channelMode != null) {
+                    updateState(channelMode.getUID(), new StringType(device.getSwitch().getMode()));
+                } else {
+                    logger.warn("Channel {} in thing {} does not exist, please recreate the thing", CHANNEL_MODE,
```

Comment:
Roll back. It didn't work. I have to revert my last commit.

Problem is that this handler receives the status of all devices in one request. We itererate the list of devices and update a specific device. We do not update the bridge itself.